CHANGELOG.md

```diff
@@ -3,14 +3,57 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## v1.0dev - [date]
+## [[1.0.0](https://github.com/sanger-tol/blobtoolkit/releases/tag/1.0.0)] – ??? - [2023-04-06]
 
-Initial release of nf-core/blobtoolkit, created with the [nf-core](https://nf-co.re/) template.
+### Enhancements & fixes
 
-### `Added`
+Initial release of sanger-tol/blobtoolkit :tada:
 
-### `Fixed`
+- Required input: (1) Tree of Life organism ID (ToLID, --input) or (2) FASTA genome file (--fasta) and `samplesheet.csv` (--input) containing paths to aligned reads in CRAM format, see description of _Full samplesheet_ in `docs/usage.md`.
+- Lineages to run BUSCO are obtained using [goat_taxonsearch module](https://nf-co.re/modules/goat_taxonsearch) instead of reading them from `config.yaml` file (--taxon: NCBI taxonomy ID or species binomial name).
```

Comment:
```suggestion
- Lineages to run BUSCO are obtained using the [Genomes on a Tree (GoaT) API](https://goat.genomehubs.org/api) from the `--taxon` parameter (NCBI taxonomy ID or species binomial name) instead of reading them from `config.yaml` file.
```

Reply:
I thought you said there was an option to provide the lineages in a file ?

Reply:
Yes `taxa_file`: https://github.com/sanger-tol/blobtoolkit/blob/dev/modules/nf-core/goat/taxonsearch/main.nf
I am going to test this a bit more with all the new changes and see how it is working.